package.json

```diff
@@ -22,6 +22,7 @@
     "test:compat": "scripts/test-compat.sh",
     "test:unit": "npm run build:test && npm run test:unit:only",
     "test:unit:only": "mocha-webpack --webpack-config test/setup/webpack.test.config.js test/specs --recursive --require test/setup/mocha.setup.js",
+    "test:unit:only:dev": "cross-env TARGET=dev yarn run test:unit:only",
```

Comment:
The rest of the scripts use `npm` here, maybe stick with that (or change them all to `yarn`)?

Reply:
You are right. Want me to change them to `yarn` or we stick with npm?

Reply:
Hm. I think it's safe to move everything to `yarn`. Up to you, from my point of view seems fine to do it here.